Remove WiringPi dependency

abouillot opened the pull request:
Remove wiringpi dependency by directly using /sys/class/gpio commands ato toggle PIN 22 - GPIO 25

abouillot committed:
Merge pull request #1 from ttn-zh/spi

Rebase to latest ttn-zh version

abouillot committed:
Change GPIO method

Use /sys/class/gpio instead of GPIO coming with wiringpi

abouillot committed:
Remove wiring pi dependency

As wiring pi is not used to reset the board

abouillot committed:
Adjust GPIO for PIN 22

PIN 22 is GPIO25. Adjusted start's reset part accordingly

gonzalocasas commented on install.sh:
```diff
@@ -96,14 +96,6 @@ INSTALL_DIR="/opt/ttn-gateway"
 if [ ! -d "$INSTALL_DIR" ]; then mkdir $INSTALL_DIR; fi
 pushd $INSTALL_DIR
 
-# Remove WiringPi built from source (older installer versions)
```
This removal part is still needed because we need to keep the already installed gateways clean

gonzalocasas commented on install.sh:
```diff
@@ -136,7 +128,6 @@ popd
 
 # Install dependencies
```
Please remove the comment and `echo` now that no dependencies are needed anymore

gonzalocasas commented on start.sh:
```diff
@@ -1,13 +1,15 @@
-sleep 0.1
-gpio -1 write 22 0
-sleep 0.1
+pin=25
```
could you please use all upper case for the variable and a more descriptive name, like `SX1301_RESET_BCM_PIN` (25 is 22 in BCM2835 denomination, right?)

gonzalocasas commented on start.sh:
```diff
@@ -1,13 +1,15 @@
+echo "1"   > /sys/class/gpio/gpio$pin/value 
+sleep 0.1  
+echo "0"   > /sys/class/gpio/gpio$pin/value
+echo "$pin"  > /sys/class/gpio/unexport 
```
please add a `sleep 0.1` between the last `0` and the `unexport` echos

gonzalocasas commented:
Really cool stuff! Thanks @abouillot for all the PRs! I added some feedback, once addressed, I will merge! :+1:

abouillot committed:
Integrate PR comments

Keep the removal of WiringPi to have all gateways on par in term of installed software
Remove the dependencies stub

abouillot committed:
Change Reset PIN name 

Change the pin variable name 
added a sleep step

abouillot commented:
Comments are now integrated
